src/main/java/com/shopify/model/ShopifyShippingLine.java

```diff
@@ -47,7 +47,7 @@ public class ShopifyShippingLine {
 	@XmlElement(name = "validation_context")
 	private String validationContext;
 	@XmlElement(name = "custom_tax_lines")
-	private String customTaxLines;
+	private List<ShopifyTaxLine> customTaxLines;
```

Comment:
Can there be a case, when customTaxLines just send a string instead of a List?

Reply:
So far, it's been null for this customer.
Additionally, there is another field called taxLines, where we use the same object.